README.rst

```diff
@@ -42,7 +42,7 @@
    :target: https://codecov.io/github/einsteinpy/einsteinpy?branch=master
 
 .. |appveyor| image:: https://img.shields.io/appveyor/ci/shreyasbapat/einsteinpy.svg?logo=appveyor&style=flat-square
-   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy
+   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy/master/branch
```

Comment:
No change was required here. The change was required in `.. |appveyor| image::` line .

Kindly see `AppVveyor branch` on [this](https://shields.io/category/build) link and update the image url.

Reply:
Did the same.